### PR TITLE
Initial draft-02 schema proposal.

### PR DESCRIPTION
**Pull request by jeremiahsmall**
Moved to carafe-fm#4

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -38,31 +44,113 @@ const cli = meow(
...
+            type: 'text',
+            name: 'bundleVersion',
+            message: 'Bundle Version?',
+            initial: '0.0.1',
+            validate: bundleVersion => carafeSemver(bundleVersion) ? true : `Not a valid version. Use MAJOR.MINOR.PATCH`
+        }
+    ]
 
-const root = path.resolve(name);
+    const response = await prompts(questions);
```
Comment (jeremiahsmall) [inline cli.js:106]: Add user prompts to initializer

File: cli.js [hunk 2]
```diff
@@ -38,31 +44,113 @@ const cli = meow(
...
+        return;
+    }
+
+    const meta = JSON.parse(fs.readFileSync(metaPath));
+    meta.id = uuidv4();
+    meta.name = bundleName;
+    meta.version = response.bundleVersion;
+    meta.category = response.bundleCategory ? response.bundleCategory : 'Development';
+    meta.creator = response.bundleCreator;
+    fs.writeFileSync(metaPath, JSON.stringify(meta, null, 4));
```
Comment (jeremiahsmall) [inline cli.js:123]: Update the meta data with prompts responses.

File: template/src/template.carafe [hunk 3]
```diff
@@ -1,123 +1,16 @@
...
-<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
-        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
-        crossorigin="anonymous"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
-        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
-        crossorigin="anonymous"></script>
+<body onload='bold("message")'>
+<div>__foo__</div>
+<div id="message">__bar__</div>
+<pre>__baz__</pre>
```
Comment (jeremiahsmall) [inline template/src/template.carafe:14]: Simplify and update starter template

**Review by DASPRiD — APPROVED** [review]

**Comment by jeremiahsmall** [comment]
Moved to https://github.com/carafe-fm/carafe-bundle-create/pull/4

**Pull request closed without merging**